### PR TITLE
BACK-318: expire pagination cursor in 5 mins

### PR DESCRIPTION
**Pull request by luchenyuxx**
This PR fix the serious issue about the memory consumption which is the cause of many issues from client side.
For the detailed explanation about the issue, please check BACK-318.

### CODE REVIEW

File: src/main/scala/co/ledger/wallet/daemon/models/GenCache.scala [hunk 1]
```diff
@@ -9,10 +11,6 @@ trait GenCache {
...
   type Cache[K, V] = concurrent.Map[K, V]
 
-  def newCache[K, V](initialCapacity: Int): Cache[K, V] = new ConcurrentHashMap[K, V](initialCapacity).asScala
-
-  val INITIAL_ACCOUNT_CAP_PER_WALLET: Int = 1000
-  val INITIAL_POOL_CAP_PER_USER: Int = 50
-  val INITIAL_OPERATION_CAP: Int = 1000
-  val INITIAL_WALLET_CAP_PER_POOL: Int = 100
+  def newCache[K <: AnyRef, V <: AnyRef]: Cache[K, V] =
+    CacheBuilder.newBuilder().expireAfterAccess(Duration.ofMinutes(5)).build[K, V]().asMap().asScala
```
[the remark this answers is not in the document]
Comment (luchenyuxx) [reply src/main/scala/co/ledger/wallet/daemon/models/GenCache.scala]: Temporal bounding will make things much better and can be sufficient in this case. Unless we can process more than 10M get operation requests per five minutes.
I thought of size bound, but couldn't get a good one.

[the remark this answers is not in the document]
Comment (luchenyuxx) [reply src/main/scala/co/ledger/wallet/daemon/models/GenCache.scala]: I'll try to add a config for ttl

**Comment by luchenyuxx** [comment]
@NajlaAmdouni Could you help me on testing this PR? I need a script to create an account which has at least 1 operation.

**Comment by luchenyuxx** [comment]
@dud225 wants this fix to be deployed as soon as possible, since we are having issues on prod.
@RomanWlm Could you review again?
@YBadiss Could you organize with @dud225 to prepare the release and deployment?

**Comment by dud225** [comment]
An alarm has been triggered last night because the CC WD instance got out of memory. See #alerts-vault

**Pull request merged**